CHANGELOG.md

```diff
@@ -4,18 +4,25 @@ Changelog
 Development version
 -------------------
 
+The highlight of this release is the support for Shapely 2.0. This makes it possible to
```

Comment:
You might be waiting until you can lock the date in, but we should just make sure there is a heading for 0.12, rather than these being under development version
```suggestion
Version 0.12 (October xx, 2022)
-------------------------------

The highlight of this release is the support for Shapely 2.0. This makes it possible to
```